devops/ansible/examples/girder-hardened/site.yml

```diff
@@ -0,0 +1,97 @@
+---
+
+- hosts: all
+  vars:
+    girder_update: no
+    girder_force: no
+    girder_always_build_assets: yes
+    girder_web_extra_args: "--all-plugins"
+    girder_hardened: yes
+    nginx_vhosts:
+      - listen: "8080 default_server"
+        ssl_certificate_key: "/etc/ssl/default_server/default_server.key"
+        ssl_certificate: "/etc/ssl/default_server/default_server.pem"
+        server_name: "girder-hardened"
+        extra_parameters: |
```

Comment:
Could we store this in a file and load it into `extra_parameters` using a [lookup](http://docs.ansible.com/ansible/playbooks_lookups.html)? That would be a lot cleaner than this inline string.

Additionally, I wonder if some of this code duplication (particularly for setting up the proxy) could be reduced with the [`assemble` module](http://docs.ansible.com/ansible/assemble_module.html).